helm_deploy/hmpps-activities-management/values.yaml

```diff
@@ -52,6 +52,8 @@ generic-service:
     elasticache-redis:
       REDIS_HOST: "primary_endpoint_address"
       REDIS_AUTH_TOKEN: "auth_token"
+    feature-toggles:
+      BVLS_FEATURE_MASTER_PUBLIC_PRIVATE_NOTES: "BVLS_FEATURE_MASTER_PUBLIC_PRIVATE_NOTES"
```

Comment:
This has been added to dev/preprod/prod namespaces.  However don't take my word for it!